test/entrypoint/config.yaml

```diff
@@ -0,0 +1,32 @@
+credentials:
+  users:
+    guest:
+      roles: [super]
+
+groups:
+  group-001:
+    replicasets:
+      replicaset-001:
+        instances:
+          master:
+            roles: [roles.metrics-export]
+            roles_cfg:
+              roles.metrics-export:
+                http:
```

Comment:
Using `http:` is actually a good idea since later we may support things like non-http metrics push or pull here. (For example, graphite push works through a raw socket.)

Reply:
Thank you.